### PR TITLE
copr: Add vectorize FromBase64.

### PR DESCRIPTION
**Pull request by pingyu**
Signed-off-by: pingyu shui.yu@126.com
PCP #6933 

###  What have you changed?
Add vectorize FromBase64.

###  What is the type of the changes?
- New feature (a change which adds functionality)

###  How is the PR tested?
- Unit test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?
No.

###  Does this PR affect `tidb-ansible`?
No.

###  Refer to a related PR or issue link (optional)
#5751, #6933

### CODE REVIEW

**Comment by zhongzc** [comment]
I suggest picking up challenge from #6933 first.

File: components/tidb_query_common/src/string.rs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```
Comment (zhongzc) [inline components/tidb_query_common/src/string.rs:1]: `tidb_query_shared_expr` is the more appropriate location to place these. You can update the branch then see something overlapped already exists there.

Comment (pingyu) [reply components/tidb_query_common/src/string.rs]: @zhongzc Comment addressed. PTAL~

**Pull request closed without merging**